back/app/routes.py

```diff
@@ -1,10 +1,29 @@
+
 from app import app,db 
 from flask import jsonify, request
 
+
 @app.route('/')
 def home():
     return "Hello, Flask!"
 
+
+@app.route('/create',methods = ['POST'])
+def create():
+        try:
+            data = request.json()
```

Comment:
` data = request.json() ` json object is not callable. make it ` data = request.json `